### ISSUE TITLE
NumberValidator has private fields

### ISSUE THREAD

**Issue by alocode** [magiclen/validators#2]
Hallo,
Im trying to add validators for a number field with the following code:
```rust
extern crate validators;
use validators::number::{Number, NumberValidator};

fn validate_number_field(value: i32) {
  let validator = NumberValidator {
    negative: ValidatorOption::NotAllow,
    zero: ValidatorOption::Allow,
  };
  validator.is_integer();
  // ...Magic should continue with other code
}
```
And I got the following errors:

```log
field 'negative' of struct 'validators::number::NumberValidator' is private
field 'negative' is private rustc(E0451)

field 'zero' of struct 'validators::number::NumberValidator' is private
field 'zero' is private rustc(E0451)
```
My Cargo.toml file dependencies:
```toml
[dependencies]
iui = { git = "https://github.com/rust-native-ui/libui-rs.git" }
diesel = { version = "1.0.0", features = ["sqlite"] }
dotenv = "0.9.0"
validators = "0.20.3"
```
I gave a quick look to the source code of Number mod and I found that those fields are indeed private.
I think I've read pretty much everything in the documentation and couldn't find a solution, am I missing something about the NumberValidator usage?

**Comment by magiclen** [comment]
You are right. I forget to add the `pub` keyword for those fields.

**Comment by magiclen** [comment]
It has been fixed in 0.20.4.

**Comment by alocode** [comment]
Thank you! Quicker than a lightning
